Resources/Prototypes/Roles/Jobs/Command/centcom_official.yml

```diff
@@ -15,12 +15,12 @@
   id: CentcomGear
   equipment:
     jumpsuit: ClothingUniformJumpsuitCentcomOfficial
-    shoes: ClothingShoesBootsJack
+    shoes: ClothingShoesBootsCombatFilled
     head: ClothingHeadHatCentcom
     eyes: ClothingEyesGlassesSunglasses
     gloves: ClothingHandsGlovesColorBlack
     outerClothing: ClothingOuterArmorBasic
-    id: CentcomPDA
-    ears: ClothingHeadsetAltCommand
-    pocket1: PaperOffice
+    id: CentcomIDCard
```

Comment:
Why

Reply:
youll need to make a centcom pda with a legit one in it then